assets/src/edit-story/theme.js

```diff
@@ -51,6 +51,7 @@ const theme = {
       v8: '#2A3140',
       v9: '#232636',
       v10: '#44485B',
+      v11: '#F2F2F2',
```

Comment:
No need to create a new value for a pure gray. This is just white/black with relevant opacity:

```js
import { rgb } from 'polished';

const Foo = styled.div`
  color: ${({ theme }) => rgb(theme.colors.fg.v1, 0.95)}; // as: 0xf2 / 0xff ~ 0.95
`;
```

Reply:
Fixed. Thank you